packages/react/popper/src/Popper.tsx

```diff
@@ -249,12 +248,12 @@ const PopperContent = React.forwardRef<PopperContentElement, PopperContentProps>
     // we need to re-compute the positioning once the parent has finally been placed.
     // https://github.com/floating-ui/floating-ui/issues/1531
     useLayoutEffect(() => {
-      if (isRoot && isPlaced) {
+      if (isRoot && isPositioned) {
         Array.from(positionUpdateFns)
           .reverse()
           .forEach((fn) => requestAnimationFrame(fn));
       }
-    }, [isRoot, isPlaced, positionUpdateFns]);
+    }, [isRoot, isPositioned, positionUpdateFns]);
```

Comment:
This should no longer be necessary since `@floating-ui/dom@1.0.5`

Reply:
Done